### DISCUSSION TITLE
WHY ?¿Made for developers, and if you are a merchant, you shouldn't download it

### DISCUSSION THREAD

**Discussion by alfonsolr09** [Oksydan/falcon#367] [General]
### Summary

Made for developers, and if you are a merchant, you shouldn't download it 

WHY??

**Comment by Oksydan** [comment]
Hi,

The installation process can be too overwhelming for merchants who are not familiar with CLI tools and code.
That's why this annotation exists in readme. The purpose of this theme was to create the best starter theme possible with a lot of good tool that help developers build and maintain projects.

**Reply by alfonsolr09** [reply]
And like merchant, Is there any possibility of being able to use the theme?

**Reply by Oksydan** [reply]
Ofc you can if you have a programming knowledge. If you don’t hire someone.

**Reply by alfonsolr09** [reply]
I can't use the theme as it comes by default?

**Reply by Oksydan** [reply]
Ofc you can but I can't guarantee that everything will be working as expected w/o setting up publicPath and compile step for example.

**Reply by alfonsolr09** [reply]
That's why I say, couldn't you compile the theme and launch a release?

**Reply by Oksydan** [reply]
Webpack publicPath is dependant on you environment configuration to load resources properly. I ship this theme compiled with default publicPath set to `/themes/falcon/assets/` but I might be different depending on configuration and theme name.
